libs/as-sdk/assembly/bytes.ts

```diff
@@ -11,7 +11,7 @@ export class Bytes {
     type: string = "hex";
     value: Uint8Array;
 
```

Comment:
Just a thought, maybe we should give this a toUtf8String() function for usage with HttpResponse